app/helpers/request_info_helper.rb

```diff
@@ -1,122 +1,305 @@
-      sort = {:isFilteredBy => true, :sortDirection => edit["#{type}_sortdir".to_sym] == 'ASC' ? 'ASC' : 'DESC'}
+    if edit[header_keys[:columns]][index] == edit[header_keys[:sort_col]]
+      sort = {:isFilteredBy => true, :sortDirection => edit[header_keys[:sort_dir]] == 'ASC' ? 'ASC' : 'DESC'}
     end
     sort
   end
 
   def none_cells(count)
-    Array.new(count) { cell_data(" ") }.unshift(cell_data("<#{_('None')}>"))
+    Array.new(count) { prov_cell_data(" ") }.unshift(prov_cell_data("<#{_('None')}>"))
   end
 
   def prov_vm_grid_cells(data, edit)
     cells = [
-      cell_data(data.name),
-      cell_data(h(data.operating_system.try(:product_name))),
-      cell_data(h(data.platform)),
-      cell_data(h(data.cpu_total_cores)),
-      cell_data(h(number_to_human_size(data.mem_cpu.to_i * 1024 * 1024))),
-      cell_data(h(number_to_human_size(data.allocated_disk_storage))),
+      prov_cell_data(data.name),
+      prov_cell_data(h(data.operating_system.try(:product_name))),
+      prov_cell_data(h(data.platform)),
+      prov_cell_data(h(data.cpu_total_cores)),
+      prov_cell_data(h(number_to_human_size(data.mem_cpu.to_i * 1024 * 1024))),
+      prov_cell_data(h(number_to_human_size(data.allocated_disk_storage))),
     ]
-    cells.push(cell_data(h(data.deprecated ? _("true") : _("false")))) if edit[:vm_headers].key?('deprecated')
+    cells.push(prov_cell_data(h(data.deprecated ? _("true") : _("false")))) if edit[:vm_headers].key?('deprecated')
     ext_name = data.ext_management_system ? h(data.ext_management_system.name) : ""
-    cells.push(cell_data(ext_name))
-    cells.push(cell_data(h(data.v_total_snapshots)))
+    cells.push(prov_cell_data(ext_name))
+    cells.push(prov_cell_data(h(data.v_total_snapshots)))
     if edit[:vm_headers].key?('cloud_tenant')
-      cells.push(cell_data(h(data.cloud_tenant ? data.cloud_tenant.name : _('None'))))
+      cells.push(prov_cell_data(h(data.cloud_tenant ? data.cloud_tenant.name : _('None'))))
     end
     cells
   end
 
   def prov_host_grid_cells(data, options)
     options[:host_columns].map do |col|
-      cell_data(h(data.send(col)))
+      prov_cell_data(h(data.send(col)))
     end
   end
+
+  def prov_pxe_img_cells(data)
+    [
+      prov_cell_data(h(data.name)),
+      prov_cell_data(h(data.description))
+    ]
+  end
+
+  def prov_iso_img_cells(data)
+    [
+      prov_cell_data(h(data.name))
+    ]
+  end
+
+  def prov_ds_grid_cells(data, edit)
+    cells = []
+    edit[:ds_columns].each do |col|
+      cells += if %w[free_space total_space].include?(col)
+                 prov_cell_data(h(number_to_human_size(data.send(col), :precision => 1)))
+               else
+                 prov_cell_data(h(data.send(col)))
+               end
+    end
+    cells
+  end
+
+  def prov_template_cells(data)
+    [
+      prov_cell_data(h(data.name)),
+      prov_cell_data(h(data.description)),
+      prov_cell_data(h(data.updated_at)),
+    ]
```

Comment:
Using `h()` here may cause problems later with javascript (for example https://github.com/ManageIQ/manageiq-ui-classic/issues/8477). The same applies for everywhere else in this file you're using `h()`

Reply:
For reference I edited them to be `h("< text & text >")` and this is how the react renders it

![Screen Shot 2022-11-15 at 11 52 43 AM](https://user-images.githubusercontent.com/29209973/201979087-0624c0c5-72eb-46a8-998a-3f5f3d493221.png)